### PR TITLE
Feature/movie details

### PR DESCRIPTION
**Pull request by Gabby-Recny**
### What (if any) features are you implementing?
- Add a method to select movie on click
### What (if anything) did you refactor?
- n/a
### Were there any issues that arose?
- Console log is slower than state change. Comment in App where setState called.
### Is there anything that you need from your teammate?
- N/A
### Refactor checklist
- [ X ] Check variable and parameter names: are they descriptive and unique?
- [ X ] Is the code DRY? SRP?
- [ X ] Is it easy for others to read your code? (explanatory comments needed?)
- [ X ] Is the syntax consistent? (e.g. use of semicolons, blank lines)
- [ X ] Are commit messages formatted correctly?

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -4,18 +4,32 @@ import './App.css';
...
+    this.state = {
+      movies: [],
+      selectedMovie: {},
+      isSelected: false,
+    };
+  }
+  componentDidMount = () => {
+    this.setState({movies: movieData.movies})
+  }
+  onClick = (id) => {
```
Comment (anewb87) [inline src/App.js:21]: Oops! We will change the name of this.

**Review by jasmyn2244 — APPROVED** [review]
Looks great! Proud of us and how we problem solved that!

File: src/App.js [hunk 2]
```diff
@@ -4,18 +4,32 @@ import './App.css';
...
   constructor() {
     super()
-    this.state = { movies: movieData.movies };
+    this.state = {
+      movies: [],
+      selectedMovie: {},
+      isSelected: false,
+    };
+  }
+  componentDidMount = () => {
```
Comment (anewb87) [inline src/App.js:18]: Put this in since we know we're going to have to use for the API calls

File: src/Cards.js [hunk 3]
```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import './Cards.css';
 
-const Cards = ({ title, posterPath, releaseDate, avgRating, id }) => {
+const Cards = ({ title, posterPath, releaseDate, avgRating, id, chooseMovie }) => {
     return (
-        <section className='movie-cards'>
+        <section className='movie-cards' id={id} onClick={() => chooseMovie(id)}>
```
Comment (anewb87) [inline src/Cards.js:6]: It works!

**Review by anewb87 — APPROVED** [review]
We have some functionality! Now we have to connect that to what we need to show.

**Pull request merged**